### PR TITLE
DLS-10416 Check your details at the start

### PR DESCRIPTION
**Pull request by LAJW**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosavefrontend/views/register/create_account.scala.html [hunk 1]
```diff
@@ -35,10 +35,6 @@
   <h1 class="govuk-heading-l">@title</h1>
 
   <p class="govuk-body">@Html(Messages("hts.register.create_account.intro"))</p>
-  <h2 class="govuk-heading-m">@messages("hts.register.create_account.your-details")</h2>
```
Comment (johno1985) [inline app/uk/gov/hmrc/helptosavefrontend/views/register/create_account.scala.html:38]: does this entry need removing from the messages files?

File: app/uk/gov/hmrc/helptosavefrontend/controllers/CheckYourDetailsController.scala [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  checkYourDetails: check_your_details
+)(
+  implicit
+  val transformer: NINOLogMessageTransformer,
+  val frontendAppConfig: FrontendAppConfig,
+  ec: ExecutionContext
+) extends BaseController(cpd, mcc, errorHandler, maintenanceSchedule) with HelpToSaveAuth with EnrolmentCheckBehaviour
+    with Logging {
+
+  def checkYourDetails: Action[AnyContent] =
```
Comment (johno1985) [inline app/uk/gov/hmrc/helptosavefrontend/controllers/CheckYourDetailsController.scala:50]: do we not need some unit tests for this new controller?

**Review by johno1985 — COMMENTED** [review]
comments added

File: app/uk/gov/hmrc/helptosavefrontend/controllers/CheckYourDetailsController.scala (hunk 2 above)
Comment (LAJW) [reply app/uk/gov/hmrc/helptosavefrontend/controllers/CheckYourDetailsController.scala:50]: We do, as I mentioned in the standup and in the post on Slack.

**Comment by johno1985** [comment]
Sort of approved, ready to QA. Unit tests to be added later prior to final approval and merging

**Comment by hmrc-web-operations** [comment]
Thank you for submitting a pull request on this HMRC repository. A member of the HMRC team will review your changes and action accordingly.

HMRC reviewer: please reference [this runbook](https://github.com/hmrc/build-and-deploy/blob/main/runbooks/reviewing_external_pull_requests.md) for guidance on how to proceed.

**Pull request closed without merging**